README.md

```
@@ -1,129 +1,202 @@
-    <h1>${{\color{#6559EA}\textsf{Funda}}}$</h1>
-    <h3>💡 즐거운 학습 경험을 통해 개발자의 지속적인 성장을 돕습니다</h3>
-    <br/>
-    
-**Funda**는 CS 지식 학습을 게임처럼 재미있게 바꾸는 웹 기반 학습 플랫폼입니다.<br/>
-체계적인 로드맵과 다양한 퀴즈 형식, 게이미피케이션 요소로 지루한 공부를 즐거운 습관으로 만들어갑니다.<br/>
-자동 복습 시스템(SRS)과 AI 기반 학습 지원으로 개인 맞춤형 학습 경험을 제공합니다.
+
+  <img src="https://github.com/user-attachments/assets/a063f0bf-37f6-49fc-99b5-15f34c39210f" alt="Funda 히어로 UI" />
+
+  <p>
+    <a href="https://funda.website"><strong>🌐 배포 사이트</strong></a> • 
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)